plugins/workspace-backpack/README.md

```diff
@@ -30,6 +41,35 @@ const backpack = new Backpack(workspace);
 backpack.init();
 ```
 
+### Configuration
+This plugin takes an optional configuration object.
```

Comment:
I think we should explain that you can leave any of these out of the object to use the default